scripts/hudson/narayana.sh

```diff
@@ -523,8 +524,15 @@ function lra_tests {
   ./build.sh install -pl ArjunaCore/arjunacore,rts -am -DskipTests -Pcommunity "$@"
   # we can't use 'mvn -f' option beacuse of Thorntail plugin issue THORN-2049
   cd ./rts/lra/
+  echo "#0. Running LRA tests using $ARQ_PROF profile"
   PRESERVE_WORKING_DIR=true ../../build.sh -fae -B -Pcommunity -P$ARQ_PROF $CODE_COVERAGE_ARGS $ENABLE_LRA_TRACE_LOGS -Dlra.test.timeout.factor="${LRA_TEST_TIMEOUT_FACTOR:-1.5}" "$@"
-  [ $? = 0 ] || fatal "LRA Test failed"
+  LRA_ARQ=$?
```

Comment:
We use capitals for environment variables. Use lower case for this temporary variable otherwise it makes this complex script harder to grok, something obvious like ok or res or success or anything that's lower case and meaningful.

Reply:
done :-) thanks @mmusgrov

Reply:
You just replaced upper case with lower case, but okay thanks.